_data/Jimut123/Jimut123.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jimut Bahan Pal
+institution: Ramakrishna Mission Vivekananda Educational and Research Institute
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **66** characters.